### DISCUSSION TITLE
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".

### DISCUSSION THREAD

**Discussion by MrBretze** [Minestom/Minestom#966] [Help]
When I start the server in idea I have this warning, it's normal or ?

SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

![image](https://user-images.githubusercontent.com/5050734/164911902-f71c4009-b195-47a5-9735-7b18d51bda22.png)

**Comment by agwoodev** [comment]
Yes, this is normal. you need to add slf4j as a dependancy and use it for logging for it to go away.
